source/Nuke.Common/Tools/CloudFoundry/CloudFoundryTasks.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019 Maintainers of NUKE.
+// Distributed under the MIT License.
+// https://github.com/nuke-build/nuke/blob/master/LICENSE
+
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Nuke.Common.Tools.CloudFoundry
+{
+    public static partial class CloudFoundryTasks
+    {
+        public static async Task CloudFoundryEnsureServiceReady(string serviceInstance)
+        {
+            bool IsCreating()
+            {    
+                var commandResult = CloudFoundryGetServiceInfo(c => c
+                        .SetServiceInstance(serviceInstance))
+                    .Where(x => x.Text.StartsWith("status:"))
```

Comment:
Line 19 en 20 can be combined to `.First(x => x.Text.StartsWith("status:"))`

Reply:
fixed